<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="lblSalida">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>581</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);
font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>PYTHON</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TXTpWD">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPwd">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Validar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
